shared/js/dataDriver.js

```diff
@@ -284,6 +301,7 @@ loop.DataDriver = function() {
       }));
       this._sse.addEventListener("open", this);
```

Comment:
It seems like we should probably check for existing `_sse` and do some cleanup as it might be possible for the old EventSource to still send events. Even if we're sure there won't be new events, we should clean up as the event handlers might keep references leaking memory.. ?